CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## NEXT - TBD
 ### Fixed
+
+### Added
+
+## [0.3.9] - 2021-07-26
+### Fixed
 - FSDP: fixed metadata saving and shard consolidation for MoE cases [#746]
```

Comment:
Can you add more detail about what this will mean for users? What can they do now with this fix that they couldn't before?

Reply:
Also below for #729 can you add more description about what "when available" means? i think it is the PyTorch version 1.8 and above that allows this feature right? Can we specify that?

Should we update our docs about #729? We can open an issue for now and tackle it during our docs fixit. WDYT?

Reply:
Feel free to file an issue on the doc. For this PR, I will skip the doc update so that Sam gets the release sooner.